sky/execution.py

```diff
@@ -567,6 +567,13 @@ def exec(  # pylint: disable=redefined-builtin
             f'{colorama.Style.RESET_ALL}')
     backend_utils.check_cluster_name_not_reserved(cluster_name,
                                                   operation_str='sky.exec')
+    if isinstance(entrypoint, sky.Task):
+        resources = entrypoint.resources
+        if len(resources) > 1:
+            logger.warning(f'{colorama.Fore.YELLOW} Passing multiple resources '
+                           'to sky.exec is not supported.'
+                           f'{colorama.Style.RESET_ALL}')
+            return
```

Comment:
Should we `raise` instead?

Also, this means we are not able to `sky exec` the same task yaml that is used for launching the cluster. Why can't we do the same checking we did in `cloud_vm_ray_backend::check_resources_fit_cluster`